Refureku/Library/Include/Public/Refureku/Config.h

```diff
@@ -9,8 +9,8 @@
 
 //Version
 #define REFUREKU_VERSION_MAJOR 2
-#define REFUREKU_VERSION_MINOR 1
-#define REFUREKU_VERSION_PATCH 2
```

Comment:
Version should not change until dev is merged to master for a new library release.